src/test/java/org/opentripplanner/middleware/TestUtils.java

```diff
@@ -166,16 +166,16 @@ private static String mockOtpPlanResponse(Request request, Response response) th
     }
 
     /**
-     * Submit plan query to OTP server and construct a monitored trip from the response.
+     * Submit plan query to OTP server and return the response.
      */
-    public static MonitoredTrip constructedMonitoredTripFromOtpResponse() {
+    public static OtpDispatcherResponse sendSamplePlanRequest() {
         // Submit a query to the OTP server.
         // From P&R to Downtown Orlando
         OtpDispatcherResponse otpDispatcherResponse = OtpDispatcher.sendOtpPlanRequest(
             "28.45119,-81.36818",
             "28.54834,-81.37745"
         );
         // Construct a monitored trip from it.
```

Comment:
Comment is out of date. You could also just remove the otpDispatcherResponse variable and return directly the sendOtpPlanRequest result.

Reply:
Updated.